src/Coveo.Connectors.EasyFilePusher/ProgramArguments.cs

```diff
@@ -0,0 +1,106 @@
+﻿using CommandLine;
+
+namespace Coveo.Connectors.EasyFilePusher
+{
+    /// <summary>
+    /// Definition of the program input arguments.
+    /// </summary>
+    public class ProgramArguments
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        [Option(Required = true, HelpText = "Cloud environment: " + nameof(CloudEnvironment.Hipaa) + ", " + nameof(CloudEnvironment.Prod) + ", " + nameof(CloudEnvironment.QA) + " or " + nameof(CloudEnvironment.Dev) + ".")]
```

Comment:
Can we get a short version of the params too? It's quite verbose to write a command line with --organization. Would be nice to have the option to use -o instead of --organization for example.

Reply:
Oui, bonne suggestion!  Je viens d'ajouter des noms courts (une lettre) aux paramètres qui seront utilisés souvent i.e. ceux qui sont obligatoires.  Le "usage" sort maintenant comme ci-bas.  Tu me diras ce que tu penses de mes choix de lettres:

Coveo.Connectors.EasyFilePusher 0.0.1
Copyright © Coveo Solutions Inc

ERROR(S):
  Required option 'e, environment' is missing.
  Required option 'r, region' is missing.
  Required option 'o, organizationid' is missing.
  Required option 's, sourceid' is missing.
  Required option 'k, apikey' is missing.
  Required option 'f, folder' is missing.

  -e, --environment       Required. Cloud environment: Hipaa, Prod, QA or Dev.

  -r, --region            Required. Cloud region: UsEast1 or EuWest1.

  -o, --organizationid    Required. ID of the organization in which to push documents.

  -s, --sourceid          Required. ID of the source in which to push documents.

  -k, --apikey            Required. API key to use.

  -f, --folder            Required. Path of the local folder that contains the documents to index.

  --include               (Default: *) Wildcard expression for which matching files will be pushed. All files are pushed
                          by default.

  --recursive             (Default: true) Whether to recursively search in sub-folders for files to push. Sub-folders
                          are searched by default.

  --batchsize             (Default: 10) How many files to push per batch.

  --help                  Display this help screen.

  --version               Display version information.

Reply:
Nice!